src/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import moment from "moment";
+
+const headings = [
```

Comment:
Notice how this is outside of the SearchResults function? We should not do that. 
I would move line 15 `const SearchResults = props => {` to line 4, so we wrap all of our code in one function.